bundle_test.go

```diff
@@ -457,298 +459,899 @@ func Test_Bundle_Validate(t *testing.T) {
 	}
 }
 
-/*
-/*
-// TODO use cases tests of []bundle and [][]bundle types
-func Test_Aggregation_Validate(t *testing.T) {
+func Test_Bundles_Copy(t *testing.T) {
```

Comment:
I don't think you're really testing copy here, but i'm not certain what you're testing - that sorting by time and version give different results?

Reply:
Slices are pointers by nature. When the same list is manipulated and copy does not really work in this case, the test would fail. Since copy works as expected the lists are different because they are ordered (manipulated) in different ways.

Reply:
Good enough.